spec/src/main/asciidoc/intro.adoc

```diff
@@ -27,6 +27,13 @@ endpoints and metrics for each process adhering to the Eclipse MicroProfile stan
 This proposal does not talk about health checks. There is a separate specification for
 https://github.com/eclipse/microprofile-health[Health Checks].
 
+=== Future Direction
+
+Over the last few months the group involved in metrics has been investigating https://micrometer.io/[Micrometer].
+This investigation will lead to significant changes to MP Metrics for releases in 2021.
+What exactly will change in the specification hasn't been defined,
+but there will be a shift towards Micrometer as a solution suited to providing metrics for Site Reliability tooling.
```

Comment:
I think it would be good to mention possibility of compatibility, as it's not clear yet whether we would provide annotation or api-level compatibility between the current API and Micrometer API.  As proposed wording...


Over the last few months the group involved in metrics has been investigating https://micrometer.io/[Micrometer].
This investigation may lead to significant changes to MP Metrics for releases in 2021.
What exactly will change in the specification, and what compatibility will be maintained between the MicroProfile 3.0 and 4.x APIs and exported metrics still hasn't been determined.
The intent will be to shift towards Micrometer as a solution suited to providing metrics for Site Reliability tooling.

Reply:
Thanks for the feedback @donbourne.

I made some minor adjustments to your suggestion.